client/src/styles/mixins.scss

```diff
@@ -17,7 +17,7 @@
 }
 
 @mixin topbar-small {
-	@media (width <= 582px) {
+	@media (width <= 765px) {
```

Comment:
The old value meant the text would be wrapping way before the area grew big enough to handle it.